src/sidebar/components/HypothesisApp.js

```diff
@@ -169,7 +168,7 @@ function HypothesisApp({ auth, frameSync, settings, session, toastMessenger }) {
         'h-full min-h-full overflow-scroll',
         // Precise padding to align with annotation cards in content
         // Larger padding on bottom for wide screens
-        'p-[9px] lg:pb-16 bg-grey-2',
+        'lg:pb-16 bg-grey-2',
```

Comment:
Horizontal padding is now handled by the `container` class on the child component. The bottom padding for wider breakpoints remains here and everything looks the same after the changes.